### PR TITLE
Convert navbar controls to links VC-2298

### PR DESCRIPTION
**Pull request by nikitahl**

### CODE REVIEW

File: public/components/navbar/controls/assignLayoutControl.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+declare const window: any
+
+const AssignLayoutControl: React.FC = () => {
+  const assignLayoutTitle = localizations ? localizations.assignLayout : 'Assign Layout to...'
+  const adminLink = window && window.vcvWpAdminUrl ? window.vcvWpAdminUrl : ''
+  const assignUrl = adminLink + 'admin.php?page=vcv-headers-footers'
+
+  return (
+    <>
```
Comment (pavelthq) [inline public/components/navbar/controls/assignLayoutControl.tsx:16]: Useless react fragment

File: public/components/navbar/controls/assignLayoutControl.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+// @ts-ignore
```
Comment (pavelthq) [inline public/components/navbar/controls/assignLayoutControl.tsx:1]: Why ts-ignore? it is not needed

File: public/components/navbar/controls/assignLayoutControl.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import React from 'react'
+import { getService } from 'vc-cake'
+
+const dataManager = getService('dataManager')
+const localizations = dataManager.get('localizations')
+
+declare const window: any
```
Comment (pavelthq) [inline public/components/navbar/controls/assignLayoutControl.tsx:8]: no need to do that

File: public/components/navbar/controls/assignLayoutControl.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+const AssignLayoutControl: React.FC = () => {
+  const assignLayoutTitle = localizations ? localizations.assignLayout : 'Assign Layout to...'
+  const adminLink = window && window.vcvWpAdminUrl ? window.vcvWpAdminUrl : ''
```
Comment (pavelthq) [inline public/components/navbar/controls/assignLayoutControl.tsx:12]: change to:

```
const adminLink = window.vcvWpAdminUrl ? window.vcvWpAdminUrl : ''
```

File: public/components/navbar/controls/backendControl.tsx [hunk 5]
```diff
@@ -1,55 +1,40 @@
-/* global setUserSetting */
-import React from 'react'
+// @ts-ignore
```
Comment (pavelthq) [inline public/components/navbar/controls/backendControl.tsx:1]: I think ts-ignore can be removed

File: public/components/navbar/controls/backendControl.tsx [hunk 6]
```diff
@@ -1,55 +1,40 @@
...
   )
 
   let dataHref = PostData.vcvCustomPostType() ? PostData.adminDashboardPostTypeListUrl() : PostData.adminDashboardUrl()
   if (editorType === 'vcv_tutorials') {
     dataHref = dataManager.get('gettingStartedUrl')
   }
 
-  let wordpressDashboardButton = (
-    <span
+  let wordpressDashboardButton: ReactNode = null
```
Comment (pavelthq) [inline public/components/navbar/controls/backendControl.tsx:28]: you can remove `= null` part and instantly assign it as in line 29 (no need for useless null)

**Review by pavelthq — CHANGES_REQUESTED** [review]
Plus I recommend to LINT your typescript usage with:
```
yarn lint
```

File: public/components/navbar/controls/assignLayoutControl.tsx (hunk 3 above)
Comment (nikitahl) [reply public/components/navbar/controls/assignLayoutControl.tsx:8]: Typescript shows errors if `window` type is not set

File: public/components/navbar/controls/assignLayoutControl.tsx (hunk 2 above)
Comment (nikitahl) [reply public/components/navbar/controls/assignLayoutControl.tsx:1]: Same logic in all notifications component files (`public/components/notifications`), so I copied from there.

File: public/components/navbar/controls/assignLayoutControl.tsx (hunk 1 above)
Comment (nikitahl) [reply public/components/navbar/controls/assignLayoutControl.tsx:16]: It was there before, wouldn't dare to touch it. Will double-check it is still needed or not.

File: public/components/navbar/controls/assignLayoutControl.tsx (hunk 4 above)
Comment (nikitahl) [reply public/components/navbar/controls/assignLayoutControl.tsx:12]: ok, I think maybe even `const adminLink = window.vcvWpAdminUrl || ''`

File: public/components/navbar/controls/backendControl.tsx (hunk 5 above)
Comment (nikitahl) [reply public/components/navbar/controls/backendControl.tsx:1]: Again, the same logic as in all notifications component files (`public/components/notifications`), so I copied from there.

Comment (pavelthq) [reply public/components/navbar/controls/backendControl.tsx:1]: Not needed anymore, will need to update notifications as well.

File: public/components/navbar/controls/assignLayoutControl.tsx (hunk 2 above)
Comment (pavelthq) [reply public/components/navbar/controls/assignLayoutControl.tsx:1]: Not needed anymore, will need to update notifications as well.

File: public/components/navbar/controls/assignLayoutControl.tsx (hunk 3 above)
Comment (pavelthq) [reply public/components/navbar/controls/assignLayoutControl.tsx:8]: Need to check typescript config it should be defined already.
@sqarf @alperbuulut would be good if you will have a look as well.

Comment (nikitahl) [reply public/components/navbar/controls/assignLayoutControl.tsx:8]: It seems that is the way to go. [Global type declarations are discouraged](https://stackoverflow.com/questions/42233987/how-to-configure-custom-global-interfaces-d-ts-files-for-typescript#answer-42257742). We couldn't find a good example of how it should be handled other than creating a `global.d.ts` file and specifying window type there.

Comment (pavelthq) [reply public/components/navbar/controls/assignLayoutControl.tsx:7]: ok, need to think about it

**Review by pavelthq — APPROVED** [review]

**Pull request merged**